src/hssm/distribution_utils/dist.py

```diff
@@ -18,11 +17,13 @@
 import pytensor.tensor as pt
 from bambi.backend.utils import get_distribution_from_prior
 from pytensor.tensor.random.op import RandomVariable
-from ssms.basic_simulators.simulator import simulator
-from ssms.config import model_config as ssms_model_config
+from ssms.hssm_support import (
+    get_simulator_fun_internal,
+    validate_simulator_fun,
+)
+from ssms.hssm_support import rng_fn as _rng_fn
```

Comment:
this is a small nitpick, but I would rather rename this to something like `ssms_rng_fn`. Using `_rng_fn` is a bit generic / confusing.